e2e/stack_test.go

```diff
@@ -396,17 +283,11 @@ func ethRollupFlow(t *testing.T, stack *e2e.StackConfig) {
 	require.NoError(t, proveWithdrawalLogs.Close())
 
 	// wait for the withdrawal finalization period before sending the withdrawal finalizing tx
-	finalizationPeriod, err := stack.L2OutputOracleCaller.FinalizationPeriodSeconds(&bind.CallOpts{})
-	require.NoError(t, err)
-	time.Sleep(time.Duration(finalizationPeriod.Uint64()) * time.Second)
-
-	// get the user's balance before the withdrawal has been finalized
-	balanceBeforeFinalization, err := stack.L1Client.BalanceAt(stack.Ctx, userETHAddress, nil)
-	require.NoError(t, err)
+	require.NoError(t, stack.WaitL1(2)) // TODO why do we need to wait this long?
```

Comment:
I'm curious as well, I left a comment above asking a similar question about waiting two blocks on L1. I see that we [also wait two blocks on L2 above](https://github.com/polymerdao/monomer/pull/267/files#diff-bad29d90d386ba872f7b623291e3e7658548ce5f925bf5fe5b34b05ed8ab3da6R235) as well.